features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -62,13 +64,26 @@ data class SessionReplayConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Sets the image recording level for the Session Replay feature.
+         * If not specified all images larger than certain dimensions will be masked by default.
```

Comment:
how customer can guess what are the dimensions/threshold here?

Reply:
My concern is that if the dimensions that we use are written here then if we modify it in the future we'll need to remember to also change it here.

Reply:
Sounds better